test/e2e/multi-cluster-deployment/use-kubestellar.sh

```diff
@@ -73,9 +73,9 @@ EOF
 :
 : -------------------------------------------------------------------------
 : "Verify that manifestworks wrapping the objects have been created in the mailbox namespaces"
-: Expect twice: one header line, one for the nginx namespace, one for the nginx deployment, one for the status agent Deployment
+: Expect twice: one header line, one for the nginx-bindingpolicy Binding\'s workload, one for the status agent Deployment
```

Comment:
(A) "workload" is vague here, particularly while issue #1754 is not resolved.

(B) Why should the namespace no longer appear in these listings?

Reply:
agree about "workload" We need some agreed terminology. Workload is not really defined in KS docs.

Reply:
Ah, we since we have a change from wrapping individual objects to wrapping everything matching a BindingPolicy, that explains the change in expected count. Why is the ManifestWork for the status agent `Deployment` separate? Because it is generated by the status controller, not the transport controller

Reply:
maybe we add the term "workload" to the architecture doc?
We actually used this term in the Binding CRD to define the list of objects (cluster-scope + namespace-scope) that are propagated from center to WEC and as a result of a specific BindingPoilcy.
workload use:
https://github.com/kubestellar/kubestellar/blob/main/api/control/v1alpha1/types.go#L196 
the fact that it's mapped to a specific BindingPolicy:
https://github.com/kubestellar/kubestellar/blob/main/api/control/v1alpha1/types.go#L171-#L173

Reply:
(A) We used to have a glossary. We should do that again.

(B) We have been designing to allow use cases with overlapping "workloads". Or, put another way: using "workload" simply to mean the user's objects that KubeStellar downsyncs and upsyncs, without supposing the "workloads" are disjoint or that a "workload" has any identifier or other realization in KubeStellar.